pkg/cmd/cli/cmd/project.go

```diff
@@ -92,17 +105,31 @@ func (o ProjectOptions) Validate() error {
 }
 
 // RunProject contains all the necessary functionality for the OpenShift cli project command
-func (o ProjectOptions) RunProject() error {
+func (o ProjectOptions) RunProject(cmd *cobra.Command) error {
```

Comment:
We're trying to create a pattern where one struct holds all the options required to perform a command.  The command implementation itself builds the options up, runs a validate on them, and then calls a no-arg `run`.  That means that `projectList` that you're pulling below would more properly live in the `ProjectOptions` struct.

That said, I think we'll want to consider whether we want a different alias for `osc get projects` or something different.  See my other comment.